### PR TITLE
chronograf: do not remove pvc when uninstalling chart

### PR DESCRIPTION
**Pull request by ivanflux**
fixes https://github.com/influxdata/helm-charts/issues/234

### CODE REVIEW

**Comment by rawkode** [comment]
I don't think we should enable this be default, but I'd be happy to see it added as a flag in the values YAML

**Comment by ivanflux** [comment]
@rawcode I added a flag keepPvc to values.yaml

File: charts/chronograf/templates/pvc.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  annotations:
+ {{- if (eq "true' .Values.persistence.keepPvc) }}
+    "helm.sh/resource-policy": keep
```
Comment (rawkode) [inline charts/chronograf/templates/pvc.yaml:13]: This conditional isn't closed?

**Review by rawkode — DISMISSED** [review]

**Review by timhallinflux — APPROVED** [review]

**Comment by timhallinflux** [comment]
@ivanflux - chart version needs a bump forward...then we can merge.

**Comment by ivanflux** [comment]
@timhallinflux done

**Review by timhallinflux — DISMISSED** [review]

**Comment by timhallinflux** [comment]
@ivanflux -- linter failed with: charts/chronograf/values.yaml
  29:13     error    trailing spaces  (trailing-spaces)

can you correct this?

**Review by timhallinflux — DISMISSED** [review]

**Comment by ivanflux** [comment]
@timhallinflux I fixed the lint error.

**Comment by putz612** [comment]
Having issues deploying with a new deployment with this change in but I am not sure exactly what is the cause. 
Error: template: chronograf/charts/chronograf/templates/pvc.yaml:12:9: executing "chronograf/charts/chronograf/templates/pvc.yaml" at <eq "true" .Values.persistence.keepPvc>: error calling eq: invalid type for comparison

**Comment by timhallinflux** [comment]
@ivanflux have a look?

**Comment by rawkode** [comment]
Sorry, @putz612. Simple type mismatch slipped by 

Fix submitted

https://github.com/influxdata/helm-charts/pull/248

**Comment by putz612** [comment]
Fantastic, I will test tonight.  Thanks for the quick commit!

**Pull request merged**